docs/sites.yml

```diff
@@ -5124,3 +5124,15 @@
   built_by: Pattyrn
   built_by_url: https://twitter.com/Pattyrn4
   featured: false
+- title: Intranet Italia Day
+  main_url: https://www.ariadnedigital.it/en
+  url: https://www.intranetitaliaday.it/en
```

Comment:
Which site you want to add,

`https://www.ariadnedigital.it/en`
or
`https://www.intranetitaliaday.it/en`

Because both `main_url` and `url` should have same domain.

Reply:
The correct url is https://www.intranetitaliaday.it/en, my mistake. Thanks @yogeshkotadiya, also for spotting the subtle formatting error.